spyder/widgets/waitingspinner.py

```diff
@@ -92,9 +93,19 @@ def paintEvent(self, QPaintEvent):
             distance = self.lineCountDistanceFromPrimary(i, self._currentCounter, self._numberOfLines)
             color = self.currentLineColor(distance, self._numberOfLines, self._trailFadePercentage,
                                           self._minimumTrailOpacity, self._color)
+
+            # Calcul the scaling factor to apply to the size and thickness
```

Comment:
Did you mean `Compute` instead of `Calcul` here?

Reply:
Yes, thanks for catching that.